ast/src/analyzed/display.rs

```diff
@@ -307,10 +307,13 @@ impl<T: Display> Display for SelectedExpressions<T> {
         write!(
             f,
             "{}[{}]",
-            self.selector
-                .as_ref()
-                .map(|s| format!("{s} $ "))
-                .unwrap_or_default(),
+            {
+                let s = self.selector.to_string();
```

Comment:
Can you add a short comment why the comparison is string-based?

Reply:
done